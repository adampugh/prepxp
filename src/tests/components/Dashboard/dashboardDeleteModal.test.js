import React from "react";
import { shallow } from "enzyme";

import DashboardDeleteModal from "../../../components/Dashboard/dashboardDeleteModal";

describe("<DashboardDeleteModal />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<DashboardDeleteModal />);
        expect(wrapper).toMatchSnapshot();
    });

    // test button triggers delete function
});