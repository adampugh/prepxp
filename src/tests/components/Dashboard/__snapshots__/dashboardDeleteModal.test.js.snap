// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DashboardDeleteModal /> should match snapshot 1`] = `
<Modal
  appElement={null}
  ariaHideApp={true}
  bodyOpenClassName="ReactModal__Body--open"
  className="modal"
  closeTimeoutMS={200}
  isOpen={false}
  parentSelector={[Function]}
  portalClassName="ReactModalPortal"
  shouldCloseOnEsc={true}
  shouldCloseOnOverlayClick={true}
  shouldFocusAfterRender={true}
  shouldReturnFocusAfterClose={true}
  style={
    Object {
      "overlay": Object {
        "background": "rgba(0, 0, 0, 0.6)",
        "zIndex": 2000,
      },
    }
  }
>
  <div
    className="modal--delete"
  >
    <span>
      <i
        className="fas fa-times"
      />
    </span>
    <h2>
      Delete interview questions?
    </h2>
    <hr />
    <button
      className="btn"
      onClick={[Function]}
    >
      Yes
    </button>
    <button
      className="btn"
    >
      No
    </button>
  </div>
</Modal>
`;
